resources/migrations/000_migrations.yaml

```diff
@@ -15303,6 +15303,25 @@ databaseChangeLog:
             columnName: slug
             newDataType: varchar(254)
 
+  - changeSet:
+      id: v48.00-017
```

Comment:
whenever we want a migration id, we need to claim it on our slack channel #migrations ( i know this is not a great way to synchronize and we do open for ideas)

you need to use `v48.00-019`, 17 and 18 has  been claimed by Noah.

Reply:
Ouch, ok... great to know.
Thanks